Formula/k/kuzu.rb

```diff
@@ -0,0 +1,57 @@
+class Kuzu < Formula
+  desc "Embeddable graph database management system built for query speed & scalability"
+  homepage "https://kuzudb.com/"
+  url "https://github.com/kuzudb/kuzu/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "e81be1e94dafba5e4a6f08b7f3a530c75926740d47e3a6b198687cd132630b6d"
+  license "MIT"
+  head "https://github.com/kuzudb/kuzu.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.12" => :build
+
+  def install
+    args = %w[
+      -DAUTO_UPDATE_GRAMMAR=0
+      -DENABLE_LTO=1
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    bin.install_symlink bin/"kuzu_shell" => "kuzu"
```

Comment:
can you put that issue as code comment into here? Thanks!

Reply:
Done